src/_data/plugins/eleventy-plugin-html-validate.json

```diff
@@ -0,0 +1,5 @@
+{
+  "npm": "eleventy-plugin-html-validate",
+  "author": "matt-auckland",
```

Comment:
To the comment below, is this your Twitter handle? It seemed to 404 for me when I checked on Twitter.

Reply:
GitHub handle 🥲 will fix

Reply:
@pdehaan I've updated the PR !